extra/redisotel/metrics.go

```diff
@@ -216,6 +216,7 @@ func (mh *metricsHook) ProcessHook(hook redis.ProcessHook) redis.ProcessHook {
 		attrs = append(attrs, mh.attrs...)
 		attrs = append(attrs, attribute.String("type", "command"))
 		attrs = append(attrs, statusAttr(err))
+		attrs = append(attrs, attribute.String("cmd", cmd.FullName()))
```

Comment:
should use [`db.operation` `semconv.DBOperation`](https://pkg.go.dev/go.opentelemetry.io/otel@v1.30.0/semconv/v1.24.0#DBOperation) attribute in [semconv v1.24](https://github.com/open-telemetry/semantic-conventions/blob/v1.24.0/docs/attributes-registry/db.md), or [`db.operation.name` `semconv.DBOperationName`](https://pkg.go.dev/go.opentelemetry.io/otel@v1.30.0/semconv/v1.26.0#DBOperationName) attribute in [semconv v1.26](https://github.com/open-telemetry/semantic-conventions/blob/v1.26.0/docs/attributes-registry/db.md), 
instead of custom attribute `cmd`.